### ISSUE TITLE
module name change in biolib 0.046

### ISSUE THREAD

**Issue by robinengler** [Ecogenomics/GTDBTk#86]
Hello,
Just to let you know that the biolib python module names and locations have changed in the latest version. In order to use biolib 0.0.46, the following changes must be made to GTDB-Tk:

```
sed -i 's@biolib_lite@biolib@g' gtdbtk/*.py gtdbtk/external/*.py
sed -i 's@from ..biolib@from biolib@g' gtdbtk/*.py gtdbtk/external/*.py
sed -i 's@biolib.execute@biolib.external.execute@g' gtdbtk/*.py gtdbtk/external/*.py
sed -i 's@biolib.prodigal_biolib@biolib.external.prodigal@g' gtdbtk/external/prodigal.py
```

best regards,
Robin

**Comment by pchaumeil** [comment]
Hello Robin,
Thanks for your email.

In the next release of GTDB-Tk, We will bee moving from the external biolib library to a self contained biolib_lite package in GTDB-Tk. 
Even though those changes have been pushed to the master branch, they are not ready yet for public use and I would recommend you to use release 0.1.6 of GTDB-Tk for the time being.

Thanks,
Pierre

**Comment by pchaumeil** [comment]
Fixed in version 0.2.1
